### ISSUE TITLE
axe-core@3.3.1 and above fails to load correctly on Safari 9.1.2 and 10.1

### ISSUE THREAD

**Issue by motss** [dequelabs/axe-core#1871]
Expectation: << Describe what you expect the product to do >>

`axe-core` loads and `axe.run` is defined.

Actual: << Describe what the product actually does >>

`axe-core` fails to load due to an error occurred while executing the script which causes `axe.run` is not defined. When a11y tests run which requires `axe.run()`, tests failed due to that.

```js
  undefined is not an object (evaluating '__magic__.globalThis = __magic__')
    <unknown> at     <unknown> at /components/test/node_modules/axe-core/axe.min.js:108:0
    <unknown> at     <unknown> at /components/test/node_modules/axe-core/axe.min.js:188:0
    <unknown> at             l at /components/test/node_modules/axe-core/axe.min.js:46:0
    <unknown> at     <unknown> at /components/test/node_modules/axe-core/axe.min.js:47:0
    <unknown> at     <unknown> at /components/test/node_modules/axe-core/axe.min.js:62:0
    <unknown> at             l at /components/test/node_modules/axe-core/axe.min.js:46:0
    <unknown> at             o at /components/test/node_modules/axe-core/axe.min.js:54:0
    <unknown> at             e at /components/test/node_modules/axe-core/axe.min.js:57:0
    <unknown> at   global code at /components/test/node_modules/axe-core/axe.min.js:14902:0
```

Motivation: << Describe why you want the behavior to be changed >>

Such behavior happens on `axe-core@3.3.1` and above. It works fine loading and executing the script with `axe-core@3.3.0`.

<pre><code>
axe-core version: >= 3.3.1
axe-webdriver, extension or other integration version: NA

Browser and Assistive Technology versions

For Tooling issues:
- Node version: v8.x
- Platform:  Linux (CircleCI)
</code></pre>

**Comment by motss** [comment]
I think `axe-core` includes magical polyfill for `globalThis` which first introduces in `v3.3.1` that breaks the code while execution.

Please consider taking a look at this and get this fixed.

**Comment by straker** [comment]
Thanks for the issue. This is a duplicate of https://github.com/dequelabs/axe-core/issues/1764. The fix didn't make it into 3.4.0 but will be available in axe-coconut and the latest develop branch soon.
